webui/src/pages/repositories/repository/compare.jsx

```diff
@@ -98,8 +98,14 @@ const CompareList = ({ repo, reference, compareReference, prefix, onSelectRef, o
                             <Table borderless size="sm">
                                 <tbody>
                                 {results.map(entry => {
-                                    const leftCommittedRef = reference.id + "@"
-                                    const rightCommittedRef = compareReference.id + "@"
+                                    let leftCommittedRef = reference.id;
+                                    let rightCommittedRef = compareReference.id;
+                                    if (reference.type === 'branch') {
```

Comment:
Could be done now or in a future refactor, but this is a constant and should be defined as such. I think it's used widely so probably doesn't belong in this PR though.

Reply:
Fixed, thanks